Update README.md (#35)
vtex-apps/google-tag-manager: docs/README.md

4. Click on **Save**.

:information_source: Access the [Google Tag Manager page](https://tagmanager.google.com/)</a> and login to you account in order to find out what is your account **GTM ID**. The number your should use is the one provided by the `Container ID` column. 

### Using VTEX IO Toolbelt

1. [Install](https://vtex.io/docs/recipes/development/installing-an-app/) the `vtex.google-tag-manager@2.x` app. You can confirm that the app has now been installed by running `vtex ls` again. 
2. Access the **Apps** section in your account's admin page and look for the Google Tag Manager box. Once you find it, click on the box.
3. Fill in the `Google Tag Manager` field with your **GTM ID**. 
4. Click on **Save**.

:information_source: Access the [Google Tag Manager page](https://tagmanager.google.com/)</a> and login to you account in order to find out what is your account **GTM ID**. The number your should use is the one provided by the `Container ID` column.

After installing the app, you are ready to use your GTM as usual by accessing your account dashboard directly on the [Google Tag Manager](https://tagmanager.google.com/) page. 

## Restrictions

> In order to avoid performance problems and unforeseen behavior, our VTEX IO Google Tag Manager solution uses the native GTM **blocklist** feature. You can read more about this feature on the [Google Developer Guide](https://developers.google.com/tag-manager/devguide).

Reviewer: ```suggestion
In order to avoid performance problems and unforeseen behavior, our VTEX IO Google Tag Manager solution uses the native GTM **blacklist** feature. You can read more about this feature on the [Google Developer Guide](https://developers.google.com/tag-manager/web/restrict).
```

> We currently blocklist the `HTML` ID, which automatically blocklists all the tags, variables and triggers of the type `customScripts`. The main consequence of this blocklist is that **Custom HTML tags will not be triggered**. 

Reviewer: ```suggestion
We currently blacklist the `html` ID, which automatically blocklists all the tags, variables and triggers of the type `customScripts`. The main consequence of this blocklist is that **Custom HTML tags will not be triggered**. 
```

> :warning: The HTML blocklist is VTEX Google Tag Manager app's default. At present, it is not possible to disable it.

Reviewer: ```suggestion
:warning: The HTML blacklist is VTEX Google Tag Manager app's default. At present, it is not possible to disable it.
```

> Most of the widely used Custom HTML tags are integrations with third party services, like Customer Chat, Analytics, Remarketing and Pixel tags. If your store needs a Custom HTML for one of those cases, the integration should be transformed into a VTEX IO Pixel App. If one does not yet exist, you can request the Pixel App creation on the VTEX IO [Store Discussion](https://github.com/vtex-apps/store-discussion) board.

Reviewer: Add the list of pixel apps available
https://github.com/topics/vtex-pixel
